diff --git a/data/io.github.TransmissionRemoteGtk.appdata.xml.in b/data/io.github.TransmissionRemoteGtk.appdata.xml.in
index 63cebac..90fb78d 100644
--- a/data/io.github.TransmissionRemoteGtk.appdata.xml.in
+++ b/data/io.github.TransmissionRemoteGtk.appdata.xml.in
@@ -19,6 +19,7 @@
   <screenshots>
     <screenshot height="512" width="973" type="default">
       <image>https://community.linuxmint.com/img/screenshots/transmission-remote-gtk.png</image>
+      <caption>Main window</caption>
     </screenshot>
   </screenshots>
   <releases>
